src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -298,6 +304,22 @@ function MultiCombobox<Item>({
                     ))}
                   </Flex>
                 )}
+                {value.length > 5 && (
```

Comment:
Instead of arbitrarily choosing `5`, can we control the presence of this button through a prop?

Reply:
I think I would like this to be on by default and have some default length.

But i will add something like `canClearAllAfter: number`